### PR TITLE
fix: last col cannot be expanded

### PR DESCRIPTION
**Pull request by a-m-dev**
Last col is now expandable

https://github.com/qlik-oss/sn-pivot-table/assets/29652890/0f0b3303-cdca-42fd-9cbf-c2b88ce2c3e6

### CODE REVIEW

**Comment by cbt1** [comment]
Did you take a look at why `layoutService.size.x` did not give the correct value?

File: src/pivot-table/components/cells/DimensionValue/Container.tsx [hunk 1]
```diff
@@ -64,7 +64,7 @@ const Container = ({
         }),
         cursor: getCursor(canBeSelected),
         background: getBackground({ styleService, isCellSelected, cell, isCellLocked }),
-        zIndex: layoutService.size.x - cell.x,
+        zIndex: isLeftColumn ? undefined : layoutService.size.x - cell.x,
```
[the remark this answers is not in the document]
Comment (haxxmaxx) [reply src/pivot-table/components/cells/DimensionValue/Container.tsx:67]: what do you mean? isn't that exactly what we are doing here? Or do you mean create the `style` object and then add zIndex to it if needed?

[the remark this answers is not in the document]
Comment (a-m-dev) [reply src/pivot-table/components/cells/DimensionValue/Container.tsx:67]: I was thinking if this is the correct way or you may thinking about some side effects this approach may have

**Comment by a-m-dev** [comment]
the `layoutService.size.x` will always count the measures in that case, so we have one measure -> we get x as 1
if i add two measures -> x will become 2 and so on

**Comment by cbt1** [comment]
> the `layoutService.size.x` will always count the measures in that case, so we have one measure -> we get x as 1 if i add two measures -> x will become 2 and so on

That makes sense

**Review by cbt1 — APPROVED** [review]

File: src/pivot-table/components/cells/DimensionValue/Container.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (a-m-dev) [reply src/pivot-table/components/cells/DimensionValue/Container.tsx:67]: + AFAIK the zIndex is related to the headers, so checking if we are in left grid and then not setting the zIndex should be fine then?

[the remark this answers is not in the document]
Comment (cbt1) [reply src/pivot-table/components/cells/DimensionValue/Container.tsx:67]: Yes, no need for `z-index` here

**Pull request merged**